security/pkg/k8s/chiron/controller.go

```diff
@@ -76,10 +76,10 @@ const (
 
 // WebhookController manages the service accounts' secrets that contains Istio keys and certificates.
 type WebhookController struct {
-	// The names of the services of mutating webhooks
-	mutatingWebhookServiceNames []string
-	// The names of the services of validating webhooks
-	validatingWebhookServiceNames []string
+	// The names of the services for which Chiron manage certs
+	serviceNames []string
```

Comment:
See above. Not user friendly (2 list matching by ordinal)

Reply:
Done:  use services []string; the format of a service will be service-name.service-namespace, with '.' as the delimiter, which means the service-name and service-namespace should not contain '.'.